cda-sovd-interfaces/src/components/ecu/mod.rs

```diff
@@ -193,19 +186,23 @@ pub mod x {
                         pub id: String,
                     }
 
-                    #[derive(Debug, Serialize)]
-                    #[cfg_attr(feature = "openapi", derive(schemars::JsonSchema))]
-                    #[cfg_attr(feature = "openapi", schemars(rename = "FlashTransferResponse"))]
+                    #[derive(Debug, Serialize, schemars::JsonSchema)]
+                    #[schemars(rename = "FlashTransferResponse")]
                     pub struct Response {
                         pub id: String,
+                        #[schemars(skip)]
+                        #[serde(skip_serializing_if = "Option::is_none")]
+                        pub schema: Option<schemars::Schema>,
                     }
                 }
                 pub mod get {
                     use serde::Serialize;
 
+                    use crate::Items;
+
                     #[derive(Serialize, Clone)]
-                    #[serde(rename_all = "PascalCase")]
-                    #[cfg_attr(feature = "openapi", derive(schemars::JsonSchema))]
+                    #[serde(rename_all = "camelCase")]
```

Comment:
question: Not really related to your PR, but would it make sense to document how the case of responses should be?